migration/migration2.go

```diff
@@ -25,7 +25,7 @@ var mig2 = Migration{
 		_, err := tx.Exec(`
 			CREATE TABLE rule (
 				module          VARCHAR PRIMARY KEY,
-				name            VARCHAR NOT NULL,
+				"name"            VARCHAR NOT NULL,
```

Comment:
Is this necessary to have here quotes? If yes, maybe will be better to add quotes to other filed names as well? WDYT?

Reply:
imho in this case it is indeed needed

Reply:
Why is it needed? If it is, I agree with @bond95 to put quotes to every column name

Reply:
@bond If none of the others are, `name` is a reserved keyword in PostgreSQL. Some of the others may be too, but probably not in this context.

Reply:
@JoseLSegura We should be probably doing that always everywhere anyways. It's just a bit of a nuisance.

Reply:
I was getting strange errors (`no column "name" in relation "rule"` or something like that) before I added the quotes.

[The documentation is here.](https://www.postgresql.org/docs/8.1/sql-keywords-appendix.html)

Reply:
It's ok for me then. TY